<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>296</width>
    <height>288</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>261</width>
     <height>261</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="1">
     <widget class="QLineEdit" name="target">
      <property name="text">
       <string>127.0.0.1</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_10">
      <property name="text">
       <string>Target</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_11">
      <property name="text">
       <string>Username</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="user">
      <property name="text">
       <string>root</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Cores</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="cores">
      <property name="text">
       <string>4</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_12">
      <property name="text">
       <string>Protocol</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QComboBox" name="protocol">
      <item>
       <property name="text">
        <string>ssh</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>mysql</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_13">
      <property name="text">
       <string>Port</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QPushButton" name="open_dictionnary">
      <property name="text">
       <string>Open Dictionnary</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLineEdit" name="password_file"/>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="port">
      <property name="text">
       <string>22</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="label_14">
      <property name="text">
       <string>Network checks</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QComboBox" name="network_check">
      <item>
       <property name="text">
        <string>yes</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>no</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QPushButton" name="quit">
      <property name="text">
       <string>Quit</string>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QPushButton" name="RunAttack">
      <property name="text">
       <string>Run Attack</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
